etc/st2packgen/slack_api_gen.py

```diff
@@ -46,10 +47,7 @@ def doublequoted_string_representer(dumper, data):
         method_url = "%s/%s" % (base_url, method)
         method_page = urlopen(method_url)
         method_soup = BeautifulSoup(method_page, 'html5lib')
-        method_description = method_soup.find('section', attrs={
-            "class": "tab_pane selected clearfix large_bottom_padding"}) \
-            .find_all('p')[0].text
-        method_description = re.sub('\n|\r', ' ', method_description)
+        method_description = ("This action is auto-generated. See %s") % method_url
```

Comment:
I'm very much not a fan of this. This forces our users to read additional documentation outside of ST2, and makes all of the action descriptions less useful for them.

Instead of replacing the description string, I would add the auto-generated part onto the old description. That way if the description is not descriptive enough, the user is at least pointed to the action's page in the Slack documentation, but if it is descriptive enough they don't have to go anywhere else.